packages/plugin-ext/src/hosted/node/scanners/scanner-theia.ts

```diff
@@ -177,6 +180,15 @@ export class TheiaPluginScanner implements PluginScanner {
             console.error(`Could not read '${rawPlugin.name}' contribution 'grammars'.`, rawPlugin.contributes!.grammars, err);
         }
 
+        try {
+            if (rawPlugin.contributes!.customEditors) {
```

Comment:
nitpick: What we really want is `?.` not `!.`

Reply:
Fixed